pytests/test_datetime.py

```diff
@@ -0,0 +1,166 @@
+    # a fixed offset as the timezone info
+    try:
+        from zoneinfo import ZoneInfo
+
+        start_at = datetime.now(ZoneInfo("US/Pacific"))
+        # Another representation of the same date
+        start_at_1 = start_at.astimezone(ZoneInfo("Europe/Rome"))
+    except ImportError:
+        start_at = datetime.now(tz=timezone(timedelta(hours=-8)))
+        # Another representation of the same date
+        start_at_1 = start_at.astimezone(tz=timezone(timedelta(hours=2)))
+
+    assert start_at == start_at_1
+
+    clock_config = TestingClockConfig(item_incr=timedelta(seconds=4), start_at=start_at)
+
+    # Now try the flow with the two different dates, we should have the same result:
+    window_config_1 = TumblingWindowConfig(
+        length=timedelta(seconds=10), start_at=start_at
+    )
+    window_config_2 = TumblingWindowConfig(
+        length=timedelta(seconds=10), start_at=start_at_1
+    )
+
+    out_1 = []
+    flow_1 = Dataflow()
+    build_flow(flow_1, window_config_1, out_1)
+
+    out_2 = []
+    flow_2 = Dataflow()
+    build_flow(flow_2, window_config_2, out_2)
+
+    run_main(flow_1)
+    run_main(flow_2)
+
+    assert len(out_1) == len(out_2)
+    assert out_1 == out_2
+
+
+def test_system_clock_config_tzinfo():
+    def gen(ws, wi, rs):
+        yield from [
+            (None, {"user": "a", "type": "login"}),
+            (None, {"user": "a", "type": "post"}),
+            (None, {"user": "a", "type": "post"}),
+            (None, {"user": "b", "type": "login"}),
+            (None, {"user": "a", "type": "post"}),
+            (None, {"user": "b", "type": "post"}),
+            (None, {"user": "b", "type": "post"}),
+        ]
+
+    def extract_id(event):
+        return (event["user"], event)
+
+    def build():
+        return defaultdict(lambda: 0)
+
+    def count(results, event):
+        results[event["type"]] += 1
+        return results
+
+    def build_flow(flow, window_config, out):
+        flow.input("inp", ManualInputConfig(gen))
+        flow.map(extract_id)
+        flow.fold_window("sum", clock_config, window_config, build, count)
+        flow.capture(TestingOutputConfig(out))
+
+    clock_config = SystemClockConfig()
+
+    # In python>=3.9 we have the ZoneInfo package, otherwise we use
+    # a fixed offset as the timezone info
+    try:
+        from zoneinfo import ZoneInfo
+
+        start_at = datetime.now(ZoneInfo("US/Eastern"))
+    except ImportError:
+        start_at = datetime.now(tz=timezone(timedelta(hours=-6)))
+
+    window_config_1 = TumblingWindowConfig(
+        length=timedelta(seconds=2), start_at=start_at
+    )
+    window_config_2 = TumblingWindowConfig(length=timedelta(seconds=2))
+
+    out_1 = []
+    flow_1 = Dataflow()
+    build_flow(flow_1, window_config_1, out_1)
+
+    out_2 = []
+    flow_2 = Dataflow()
+    build_flow(flow_2, window_config_2, out_2)
+
+    run_main(flow_1)
+    run_main(flow_2)
+    assert len(out_1) == len(out_2)
```

Comment:
Similar questions as above test.

Reply:
Similar concept, maybe this test is reduntant, but the idea is that settings `start_at` to `now` in python, and not doing it (so setting it in rust) should give the same result, but it didn't with `pyo3-chrono` if the `now` in python had different timezone info